initial connector funding is failing in integration tests

https://github.com/interledgerjs/ilp-kit/blob/master/api/src/lib/app.js#L111
calls https://github.com/interledgerjs/ilp-kit/blob/master/api/src/lib/pay.js#L21
calls https://github.com/interledgerjs/ilp-kit/blob/master/api/src/lib/spsp.js#L63
calls https://github.com/interledgerjs/ilp/blob/master/src/lib/spsp.js#L175
which fails because the spsp details are missing.

The ilp-kit integration test logs the error:
```
ilp-kit[Wallet2] [api] 2017-04-28T13:31:04.809Z app CRIT { AssertionError: missing SPSP response in payment
ilp-kit[Wallet2] [api]     at Object.sendPayment (/app/integration-test/ilp-kit/node_modules/ilp/src/lib/spsp.js:178:3)
```
and continues, so the tests pass, but it's still broken (probably if you install ilp-kit from scratch now, the connector will not be funded).

--- a/api/src/lib/app.js
+++ b/api/src/lib/app.js
@@ -102,13 +102,16 @@ module.exports = class App {
 
     // Initial connector funding
     if (connectorAccount && connectorAccount.new) {
-      await this.pay.pay({
+      this.log.info('Funding new connector account.')
+      const quote = await this.spsp.quote({
+        destination: connectorAccount.identifier,
         user: adminAccount,
-        destination: connectorAccount.username,
-        quote: {
-          sourceAmount: 1000,
-          destinationAmount: 1000
-        },
+        sourceAmount: 1000
+      })
+
+      await this.pay.pay({
+        quote,
+        user: adminAccount,
         message: 'Initial connector funding'
       })
     }

